packages/client/components/ui/components/features/messaging/elements/MessageReply.tsx

```diff
@@ -107,6 +107,10 @@ const Link = styled("a", {
 export function MessageReply(props: Props) {
   const client = useClient();
 
+  const renderReplyContent = (content: string) => {
+    return renderSimpleMarkdown(content);
+  };
```

Comment:
We should limit the length of the content to something reasonable like ~ 128 characters since everything longer than that likely won't appear anyways, I'm not sure what a good length would be.

Reply:
Updated